gamestonk_terminal/portfolio_optimization/README.md

```diff
@@ -0,0 +1,126 @@
+# Portfolio Optimization
```

Comment:
I think overall the README can be improved, I can take care of this!

Reply:
Yeah my bad,  definitely rushed through

Reply:
No worries, I can take care of it. I'm usually really picky so I don't mind doing it